go-apps/meep-metrics-engine/server/v2/metrics-engine.go

```diff
@@ -589,6 +602,95 @@ func mePostSeqQuery(w http.ResponseWriter, r *http.Request) {
 				continue
 			}
 			t := metricTime.Format("15:04:05.000")
+
```

Comment:
This function is getting long, please try defining other functions to call from this one.

Reply:
Sure, addressed. Added a common function updateSequenceMetrics that updates the response based on the request params: mermaid or sdorg and stronly or listonly.